ui/app/AppLayouts/Profile/stores/ContactsStore.qml

```diff
@@ -118,6 +120,10 @@ QtObject {
 
     function removeTrustStatus(pubKey) {
         d.contactsModuleInst.removeTrustStatus(pubKey)
+
+        const contactDetails = Utils.getContactDetailsAsJson(pubKey, false, false, true)
+        const displayName = SQUtils.Emoji.parse(ProfileUtils.displayName(contactDetails.localNickname, contactDetails.name, contactDetails.displayName, contactDetails.alias))
+        Global.displaySuccessToastMessage(qsTr("Untrusted mark removed for %1").arg(displayName))
```

Comment:
It would be better to have the call start from the Nim side, since it might have failed.

We call the event `SIGNAL_REMOVED_TRUST_STATUS` if the action succeeded. You can hook into one of the modules that listens to that event and then call the usual signal to have a ToastMessage

Reply:
I'll have a look yeah, good idea

Reply:
@jrainville done it as you suggested, works great; can you pls give this PR a test? TIA